samples/DLLMethodCaller/DLLMethodCallerEA.mq4

```diff
@@ -41,36 +41,59 @@
 	int nquotes_get_property_array_size(string name);
 	int nquotes_get_property_adouble(string name, double& value[]);
 #import
+//extern int MoneyAmount = 12345;
+//extern datetime MyBirthday;
+//extern bool AmMarried;
 
 int init()
 {
+   EventSetTimer(5);
 	nquotes_setup("DLLMethodCaller.DLLMethodCallerEA", "DLLMethodCaller");
-	
 	return (nquotes_init());
 }
 
 int start()
 {
-	return (nquotes_start());
+   GiveMoney(10);
+   nquotes_set_property_datetime ("MyBirthday",TimeCurrent() - 1000000000);
+   nquotes_set_property_string ("ActionName","PredictFuture");
+	Comment("FuturePrediction is: " + PredictFuture());
+	GiveMoney(100);
+	nquotes_set_property_datetime ("MyBirthday",TimeCurrent() - 1000000000);
+	nquotes_set_property_string ("ActionName","PredictFuture");
+	Comment("FuturePrediction is: " + PredictFuture());
+	return 0;
 }
 
 int deinit()
 {
-	return (nquotes_deinit());
+   return (nquotes_deinit());
 }
 
 // optional event handlers:
 // https://docs.mql4.com/basis/function/events
 void OnTimer()
 {
+   start();
 	nquotes_on_timer();
 }
 
 void OnChartEvent(const int id, const long& lparam, const double& dparam, const string& sparam)
-{
-	nquotes_on_chart_event(id, lparam, dparam, sparam);
+{  
+  nquotes_on_chart_event(id, lparam, dparam, sparam);
+}
+
+string PredictFuture(){
```

Comment:
pass `datetime birthday` and `bool isMarried` here too

Reply:
Good idea.